gocwebtemplate-core/gocwebtemplate-core-base/src/main/java/goc/webtemplate/component/AbstractCoreBean.java

```diff
@@ -1452,6 +1453,24 @@ public void setCustomFooterSections(ArrayList<FooterSection> value) {
         this.customFooterSections = value;
     }
     
+    /**
+     * Returns whether the placeholder menu should be hidden while a custom menu is being loaded
+     *
+     * can be overriden programatically.
+     */
+    public boolean getHidePlaceholderMenu() {
+        return this.hidePlaceholderMenu;

```

Comment:
Abosutely need a `this.initializeOnce();` before returning the value (to give a chance to overriders to set the value)

Reply:
done